code/modules/antagonists/gang/gang.dm

```diff
@@ -354,7 +369,8 @@
 	free_clothes = list(/obj/item/clothing/under/costume/dutch,
 						/obj/item/clothing/head/bowler,
 						/obj/item/clothing/suit/dutch,
-						/obj/item/toy/crayon/spraycan)
+						/obj/item/toy/crayon/spraycan
+						/obj/item/melee/baseball_bat/ablative)
```

Comment:
Are you actually being serious

Reply:
Yeah look at my in-depth reason and the faq discussion of how I preview it playing out.

Reply:
The Dutch gang are cowboys. And katanas qdel projectiles they block. Even in the absolute lunatic internal logic you got, it's incorrect.